nethserver-samba.spec

```diff
@@ -43,7 +43,9 @@ cp -a %{name}.json %{buildroot}/usr/share/cockpit/nethserver/applications/
 cp -a api/* %{buildroot}/usr/libexec/nethserver/api/%{name}/
 
 (cd root   ; find . -depth -print | cpio -dump %{buildroot})
-%{genfilelist} %{buildroot} > %{name}-%{version}-filelist
+%{genfilelist} %{buildroot} | grep -v \
+    -e '^/etc/sudoers.d/50_nsapi_nethserver_samba' \
```

Comment:
Why not using `--file` option like here https://github.com/NethServer/nethserver-nextcloud/pull/60/files#diff-5900e529be8febe6e74c5df8a25eb27aR47 ?